fineract-provider/src/main/java/org/apache/fineract/portfolio/delinquency/service/DelinquencyWritePlatformServiceImpl.java

```diff
@@ -506,6 +526,7 @@ private void setDelinquencyDetailsForInstallment(final Loan loan, final LoanRepa
                             installmentDelinquencyData.getDelinquentAmount());
                     loanInstallmentDelinquencyTagRepository.delete(previousInstallmentDelinquencyTag);
                     // event installment delinquency range change
+                    isDelinquencyRangeChanged = true;
                 } else {
                     previousInstallmentDelinquencyTag.setOutstandingAmount(installmentDelinquencyData.getDelinquentAmount());
```

Comment:
Shall we raise an event also if the ranges are not changed just the amount?

Reply:
@adamsaghy Please comment.

Reply:
No need.

Reply:
okay